### PR TITLE
update the all-owners.sh and the ALL-OWNERS

### PR DESCRIPTION
**Pull request by Rozzii**
This commit:
  - Extends the all-owners.sh to match the maintainer's GitHub handle against the CNCF maintainer spread sheet and match the handles against the DCO signatures found in the repositories.
  - Also updates the ALL-OWNERS file according to the output of the updated script.
  - Add 4 e-mail addresses manually because the script can't find them, 2 maintainers used different names for the signatures than what is documented in the CNCF spread sheet and I have contacted them to rectify this issue. Two other maintainers haven't been active since the DCO enforcement was put in place so I had to add their mail manually
  - Add .gitignore file to exclude repository metadata files and .csv files used during the execution of the all-owners.sh The motivation behind this commit is to make the ALL-OWNERS and all-owners.sh aligned with the CNCF requirements. (Incubation)

### CODE REVIEW

**Comment by Rozzii** [comment]
/cc @smoshiur1237 @elfosardo @mboukhalfa 
Please take a look this will be needed for incubation.

File: maintainers/all-owners.sh [hunk 1]
```diff
@@ -66,15 +66,55 @@ all_owners_raw()
     done
 }
 
+# Find e-mail address of the commiter based on theire name and the DCO
```
Comment (smoshiur1237) [inline maintainers/all-owners.sh:69]:
```suggestion
# Find e-mail address of the committer based on their name and the DCO
```

**Review by Sunnatillo — COMMENTED** [review]
LGTM

File: maintainers/all-owners.sh [hunk 2]
```diff
@@ -66,15 +66,55 @@ all_owners_raw()
...
+        cd ..
+        if [[ -n "${mail}" ]]; then
+            break
+        fi
+    done
+    echo "${mail}"
+}
+
+# The idea is to match 3 sources of truth against each other, the DCO
+# signatures, the CNCF maintainer spread sheet and the OWNER and OWNERS_ALIASES
```
Comment (smoshiur1237) [inline maintainers/all-owners.sh:87]:
```suggestion
# signatures, the CNCF maintainer spread sheet, the OWNER and OWNERS_ALIASES
```

File: maintainers/all-owners.sh (hunk 1 above)
Comment (mquhuy) [reply maintainers/all-owners.sh:69]: Also, "the" in "the DCO" should be dropped.

File: maintainers/all-owners.sh [hunk 3]
```diff
@@ -66,15 +66,55 @@ all_owners_raw()
...
+        cd ..
+        if [[ -n "${mail}" ]]; then
+            break
+        fi
+    done
+    echo "${mail}"
+}
+
+# The idea is to match 3 sources of truth against each other, the DCO
+# signatures, the CNCF maintainer spread sheet, the OWNER and OWNERS_ALIASES
```
Comment (mquhuy) [inline maintainers/all-owners.sh:87]: the OWNER and OWNERS_ALIASES files

File: maintainers/all-owners.sh (hunk 1 above)
Comment (Rozzii) [reply maintainers/all-owners.sh]: Done and done.

File: maintainers/all-owners.sh (hunk 2 above)
Comment (Rozzii) [reply maintainers/all-owners.sh]: done

File: maintainers/all-owners.sh (hunk 3 above)
Comment (Rozzii) [reply maintainers/all-owners.sh]: done

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

**Comment by smoshiur1237** [comment]
@elfosardo Please approve

**Review by kashifest — COMMENTED** [review]
/approve

**Pull request merged**